.github/workflows/update-flathub.yml

```diff
@@ -0,0 +1,43 @@
+name: update-flathub
+on:
+  workflow_dispatch:
+
+jobs:
+  update-flathub:
+    environment: flathub
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Clone source repository
+        uses: actions/checkout@v2
+        with:
+          path: src
+          # we need tags for what comes next...
+          fetch-depth: 0
+
+      - name: Clone flathub repository
+        uses: actions/checkout@v2
+        with:
+          path: flathub
+          repository: cockpit-project/org.cockpit_project.CockpitClient
```

Comment:
Conceptually, why does this clone our fork instead of the actual flathub repo? Can we assume that they are always in sync? Or do we need that hack because the default github token can't create PRs on https://github.com/flathub/org.cockpit_project.CockpitClient ?

Reply:
It's not about the PRs: it's about the ability to push.  We need admin access to install deploy keys, and we don't have admin access on the repo in the flathub project.  So we push to our own fork (which we do have a deploy key for) and then propose the PR from there.

Reply:
Right, that was my idea -- check out main from the upstream repo, then create a "version" branch for the release, and push that to our fork. So that we don't have to keep our fork's "main" branch up to date, and it'd be basically irrelevant (we could even remove it)

Reply:
Ah!  This is a good point.  I didn't think of that.  And indeed, I think your proposal is the right one.  I'd probably push a main branch with a README.md (since GitHub gets grumpy when there are no branches).

I'd prefer to make this change as a follow-up though, mostly due to how difficult it is to test the branch in its current state (due to this annoying github limitation about triggering workflows from branches when a workflow with the same name doesn't exist on the main branch).

Reply:
Ack, makes sense -- deal!